tests/instruments/test_instrument.py

```diff
@@ -23,12 +23,13 @@
 #
 
 import pytest
-from pymeasure.instruments.instrument import Instrument, FakeInstrument
+from pymeasure.instruments.fakes import SwissArmyFake
+from pymeasure.instruments.instrument import Instrument
 from pymeasure.instruments.validators import strict_discrete_set, strict_range
 
 
 def test_fake_instrument():
-    fake = FakeInstrument()
+    fake = SwissArmyFake()
```

Comment:
hm, I actually would prefer to keep using the simpler `FakeInstrument` for the general tests (reduces the number of "moving parts"), and use the `SwissArmyFake` only where needed.

Reply:
Ok, yeah good point. Changing it back.